File valid_path/karma.conf.js does not exist!

### Expected behaviour
When I run karma, I get message `valid_path/karma.conf.js`, while the file it certainly there. 

### Actual behaviour

The error happens in this file:
https://github.com/karma-runner/karma/blob/f94284d4cf01b2b5c473e62fe32732cf744b1797/lib/config.js#L356

File gets required successfully, but contains an inner require which fails.
Error handler sees the error type as `MODULE_NOT_FOUND`, but assumes that it's for outer, not inner require. 

There's is actually a test for this:
` && e.message.includes(configFilePath)`

But it fails because the path is in stack trace, which is part of the message.

```Cannot find module '../../../karma.conf'
Require stack:
- valid_path/karma.conf.js
....
```

### Environment Details

- Karma version (output of `karma --version`):
Karma version: 3.1.4
- Relevant part of your `karma.config.js` file
```javascript
require('I do not exist');
```

### Steps to reproduce the behaviour
1. create `karma.conf.js`, inside do: 

```javascript
require('I do not exist');
```

2. start karma
3. Get an error saying `karma.config.js` does not exist.

--- a/lib/config.js
+++ b/lib/config.js
@@ -353,19 +353,15 @@ function parseConfig (configFilePath, cliOptions) {
         configModule = configModule.default
       }
     } catch (e) {
-      if (e.code === 'MODULE_NOT_FOUND' && e.message.includes(configFilePath)) {
-        log.error(`File ${configFilePath} does not exist!`)
-      } else {
-        log.error('Invalid config file!\n  ' + e.stack)
+      log.error('Error in config file!\n  ' + e.stack || e)
 
-        const extension = path.extname(configFilePath)
-        if (extension === '.coffee' && !COFFEE_SCRIPT_AVAILABLE) {
-          log.error('You need to install CoffeeScript.\n  npm install coffeescript --save-dev')
-        } else if (extension === '.ls' && !LIVE_SCRIPT_AVAILABLE) {
-          log.error('You need to install LiveScript.\n  npm install LiveScript --save-dev')
-        } else if (extension === '.ts' && !TYPE_SCRIPT_AVAILABLE) {
-          log.error('You need to install TypeScript.\n  npm install typescript ts-node --save-dev')
-        }
+      const extension = path.extname(configFilePath)
+      if (extension === '.coffee' && !COFFEE_SCRIPT_AVAILABLE) {
+        log.error('You need to install CoffeeScript.\n  npm install coffeescript --save-dev')
+      } else if (extension === '.ls' && !LIVE_SCRIPT_AVAILABLE) {
+        log.error('You need to install LiveScript.\n  npm install LiveScript --save-dev')
+      } else if (extension === '.ts' && !TYPE_SCRIPT_AVAILABLE) {
+        log.error('You need to install TypeScript.\n  npm install typescript ts-node --save-dev')
       }
       return process.exit(1)
     }

--- a/test/unit/config.spec.js
+++ b/test/unit/config.spec.js
@@ -118,7 +118,8 @@ describe('config', () => {
 
       expect(logSpy).to.have.been.called
       const event = logSpy.lastCall.args
-      expect(event).to.be.deep.equal(['File /conf/not-exist.js does not exist!'])
+      expect(event.toString().split('\n').slice(0, 2)).to.be.deep.equal(
+        [`Error in config file!`, `  Error: Cannot find module '/conf/not-exist.js'`])
       expect(mocks.process.exit).to.have.been.calledWith(1)
     })
 

